src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4284,6 +4318,30 @@ components:
           $ref: '#/components/schemas/ResourceLocks'
       description: >
         Complete definition of a dataset.
+    SnapshotBuilderDatasetModel:
```

Comment:
I am not sure about this naming.  When I think of a snapshot builder in the context of TDR, I think of our own snapshot builder in the TDR UI. Naming things is hard :(

Reply:
I think this was the best name that @nmalfroy @pshapiro4broad and I could come up with - because cohorts are something else (a part of a SnapshotRequest), and datasets have an existing meaning in TDR, so there's not really anything better.